include/silo/storage/column/pango_lineage_column.h

```diff
@@ -7,22 +7,27 @@
 #include <roaring/roaring.hh>
 
 #include "silo/common/pango_lineage.h"
+#include "silo/storage/column/column.h"
 
 namespace silo::storage::column {
 
-class PangoLineageColumn {
+class PangoLineageColumn : public Column {
   public:
    template <class Archive>
    [[maybe_unused]] void serialize(Archive& archive, const unsigned int /* version */) {
+      archive& values;
       archive& value_id_lookup;
+      archive& id_value_lookup;
       archive& indexed_values;
+      archive& indexed_sublineage_values;
    }
 
   private:
+   std::vector<uint32_t> values;
    std::unordered_map<common::PangoLineage, uint32_t> value_id_lookup;
+   std::vector<common::PangoLineage> id_value_lookup;
```

Comment:
```suggestion
   std::vector<common::PangoLineage> value_lookup;
```

or maybe

```suggestion
   std::vector<common::PangoLineage> value_id_to_value_lookup;
```
?

Reply:
I wanted to keep it similar to the above `value_id_lookup` but can see that it may be confusing

Reply:
Ah, my intention behind the name there was "a lookup for value ids", not a "value to id lookup" ;)